File: weknot_android/app/src/main/java/com/example/weknot_android/network/request/LoginRequest.java
package com.example.weknot_android.network.request;

import com.example.weknot_android.util.Utils;

import java.security.NoSuchAlgorithmException;

public class LoginRequest {

    private String id;

    private String pw;

    public LoginRequest(String id, String pw) {
        try {
            this.id = id;
            this.pw = Utils.encryptSHA512(pw);
        } catch (NoSuchAlgorithmException e) {
            e.printStackTrace();
        }
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getPw() {
        return pw;
    }

    public void setPw(String pw) {
        this.pw = pw;
    }
}

File: weknot_android/app/src/main/java/com/example/weknot_android/network/request/LoginRequest.kt
package com.example.weknot_android.network.request

import com.example.weknot_android.util.Utils
import java.security.NoSuchAlgorithmException

class LoginRequest(id: String, password: String) {
    lateinit var id: String
    lateinit var password: String

    init {
        try {
            this.id = id
//            this.pw = Utils.encryptSHA512(pw)
            this.password = password
        } catch (e: NoSuchAlgorithmException) {
            e.printStackTrace()
        }
    }
}